docgenerator/spec/templates/example_detail.html

```diff
@@ -19,7 +19,7 @@
 <h1>{{ example.name }}</h1>
 
 {% if example.image_url %}
-<p width="100%"><img src="{% relative_url_string example.image_url %}" width="100%"></p>
+<p><img src="{% relative_url_string example.image_url %}"/></p>
```

Comment:
We're not using XHTML for this site, so the slash at the end of <img/> isn't necessary (to be consistent with the rest of the templates).